crates/steel-core/src/steel_vm/primitives.rs

```diff
@@ -949,13 +952,116 @@ fn equality_module() -> BuiltInModule {
     module
 }
 
+/// Real numbers ordering module.
+#[steel_derive::define_module(name = "steel/ord")]
 fn ord_module() -> BuiltInModule {
     let mut module = BuiltInModule::new("steel/ord");
+
+    /// Compares two real numbers to check if the first is greater than the second.
+    ///
+    /// (> left right) -> bool?
+    ///
+    /// * left : real? - The first real number to compare.
+    /// * right : real? - The second real number to compare.
+    ///
+    /// # Examples
+    /// ```scheme
+    /// > (> 3 2) ;; => #t
+    /// > (> 1 1) ;; => #f
+    /// > (> 3/2 1.5) ;; => #f
+    /// > (> 3/2 1.4) ;; => #t
+    /// ```
+    #[steel_derive::function(name = ">")]
+    fn greater_than(left: &SteelVal, right: &SteelVal) -> Result<SteelVal> {
```

Comment:
So the only other comment I'd have is that these functions are typically multi-arity - as in they take an arbitrary number of parameters. Could we use the rest args here to implement that? That would apply to all of the primitives here as well

Reply:
Hey, I've updated the PR; One thing I noticed though is that `arty = "AtLeast(1)"` seems to do nothing and proceeds with things like `(>)`

Reply:
I believe the arity annotation only comes into play with documentation, so it won't enforce any checks

Reply:
So in this case - the only thing remaining is to add an arity check that we're passing at least one argument in (which I think is the behavior we want) - otherwise this LGTM

Reply:
Got it, updated the check

Then I think we're good to go